packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/done.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Edit2 as EditIcon } from 'react-feather';
+
+import { Price } from '@magento/peregrine';
+
+import { mergeClasses } from '../../../classify';
+import Icon from '../../Icon';
+import LoadingIndicator from '../../LoadingIndicator';
+import defaultClasses from './done.css';
+
+const editIconAttrs = {
```

Comment:
Hmm I wonder if we should generalize the `done` state cards for use by other steps or auth view. Could take a title, content and `onEdit` callback. Would prevent everyone from rewriting their own versions of a done state.

Reply:
Yep. We decided that's out of scope here but I'll do a fast follow.